readme.md

```diff
@@ -36,6 +36,7 @@
     <li><a href="https://reactjs.org/">CSS</a></li>
     <li><a href="https://reactjs.org/">JavaScript</a></li>
     <li><a href="https://reactjs.org/">React</a></li>
+     <li><a href="https://reactjs.org/">Redux</a></li>
   </ul>
```

Comment:
- [ ] Kindly ensure that these links are redirecting to appropriate page respectively, this will make your README to be more engaging and portray correct information. 👍